src/compo/viirs_l1bnc2ioda.py

```diff
@@ -170,7 +183,9 @@ def _read(self):
                 err = obsgrp.variables[errname]
                 err.set_auto_scale(False)
 
-                vals[:, ichan] = obs[:].data.ravel()
+                # NASA VIIRS stored reflectance need to divide by
+                # cosine of solar zenith angle to get true reflectance
+                vals[:, ichan] = obs[:].data.ravel() / np.cos(solar_za * deg2rad)
```

Comment:
For today, I think you should place this line within a logical/boolean IF-test and set a true/false variable near the top of the code to be `False` for right now.  The CRTM-UFO code for visible reflectance is currently NOT dividing by `cosine(solarZenithAngle)`.  At a later time, I believe we need to include a YAML option for turning on or off this division in UFO.

Reply:
@gthompsnJCSDA Thank you, I will update it.